pyproject.toml

```diff
@@ -39,6 +39,7 @@ csv2rdf = 'rdflib.tools.csv2rdf:main'
 rdf2dot = 'rdflib.tools.rdf2dot:main'
 rdfs2dot = 'rdflib.tools.rdfs2dot:main'
 rdfgraphisomorphism = 'rdflib.tools.graphisomorphism:main'
+sparqlquery = 'rdflib.tools.sparqlquery:main'
```

Comment:
Could you rename this shorter to "sq"? I've added short forms of all command line flags to make this shorter to use too

Reply:
You can leave the script as `sparqlquery.py` but just change the pyproject.toml registration of it to `sq`. I just think that will be easier to use on the command line